Issue #3619: NeedBraces need clear description of allowSingleLineStatement (#14133)
checkstyle/checkstyle: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example1.txt

    ++count;
while (value.incrementValue() < 5); // violation, empty loop body not allowed
switch (num) {

>   case 1: counter++; break;         // OK, Exception Case

Reviewer: `// OK, 'break' is not counted to allow compact view

Reviewer: Please add attention note to description https://checkstyle.org/checks/blocks/needbraces.html#Description

Author: @romani  // Ok, break would be good wording

Reviewer: Attention: break in case blocks is not counted to allow compact view

Author: > Please add attention note to description https://checkstyle.org/checks/blocks/needbraces.html#Description

Done @romani

Author: > `// OK, 'break' is not counted to allow compact view

Updated the comment

>   case 1: counter++; break;         // OK, 'break' is not counted to allow compact view

Reviewer 2: ```suggestion
  case 1: counter++; break;         // OK, 'LITERAL_CASE' is not in default tokens
```
